kubernetes/samples/charts/traefik/README.md

```diff
@@ -4,102 +4,95 @@ This sample demonstrates how to install the Traefik ingress controller to provid
 load balancing for WebLogic clusters.
 
 ## Install the Traefik operator with a Helm chart
-The Traefik Helm chart is located in the official Helm project `charts` directory at https://github.com/helm/charts/tree/master/stable/traefik.
-The chart is in the default repository for Helm.
+This document is based on Traefik version 2.x with Helm chart at https://github.com/containous/traefik-helm-chart.
+For more information about Traefik, see https://docs.traefik.io/
 
 To install the Traefik operator in the `traefik` namespace with default settings:
 ```
-$ helm install --name traefik-operator --namespace traefik stable/traefik
+$ helm repo add traefik https://containous.github.io/traefik-helm-chart
+$ helm repo update
+$ helm install traefik-operator traefik/traefik --namespace traefik
```

Comment:
these instructions do not work - says namespace traefik does not exist

Reply:
Add a command   ( $ kubectl create namespace traefik )